Probesys/bileto
# new: Allow to create and list contracts

## Related issue(s)



https://github.com/Probesys/bileto/issues/29

## Changes



- create a Contract entity + repository + migration + factory
- add 3 new permissions related to contracts + a new "salesman" role in the seeds
- add a "contract" icon
- create a page to list the contracts
- create a page to create contracts
- create a page to display a contract
- add a page to go to the tickets or the contracts of an organization

## How to test manually



- run the seeds
- login with `benedict@example.com`
- go to an organization > contracts
- create a contract → check you're redirected to the list of contracts
- open the contract → check it renders correctly
- login with `charlie@example.com` → check you can't go to the contracts page

## Checklist
